hikari/users.py

```diff
@@ -136,6 +137,32 @@ def app(self) -> traits.RESTAware:
     def avatar_hash(self) -> undefined.UndefinedNoneOr[str]:
         """Avatar hash for the user, if they have one, otherwise `builtins.None`."""
 
+    @property
+    @abc.abstractmethod
+    def banner_hash(self) -> undefined.UndefinedNoneOr[str]:
+        """Banner hash for the user, if they have one, otherwise `hikari.undefined.UNDEFINED`."""
+
+    @property
+    def banner_url(self) -> typing.Optional[files.URL]:
```

Comment:
This is inconsistent with how properties like avatar_url are declared on User due to the fact that PartialUser can't guarantee that it'll know whether they have a banner or not